bin/crew

```diff
@@ -1258,7 +1258,9 @@ def install
     Command.remove(@pkg, @opt_verbose)
   end
 
-  unless @pkg.is_fake?
+  if @pkg.is_fake?
+    puts "The #{@pkg.name} is_fake property is true. File installs will be skipped.".orange if @opt_verbose
```

Comment:
This wording is technically correct but I'm thinking it might be more human readable to say:
```
put "Install will be skipped since #{@pkg.name} is a fake package."...
```